src/server/services/procedures/alexa/types.js

```diff
@@ -159,7 +159,11 @@ const SlotTypes = [
     'AMAZON.WrittenCreativeWorkType',
 ];
 
+let _registeredTypes = false;
```

Comment:
Is this added just in case or is it actually registering the types twice?

Reply:
I'm not sure why, but `npm test` was suddenly failing there for some reason, and that fixed it